cola-tests/src/main/java/com/github/bmsantos/core/cola/utils/ColaUtils.java

```diff
@@ -78,11 +78,11 @@ public static String binaryToResource(final String binaryFormat) {
     }
 
     public static String binaryToResourceClass(final String binaryFormat) {
-        String result = binaryToResource(binaryFormat);
-        if (!isClassFile(result)) {
-            result += CLASS_EXT;
+        StringBuilder result = new StringBuilder(binaryToResource(binaryFormat));
+        if (!isClassFile(result.toString())) {
+            result.append(CLASS_EXT);
```

Comment:
Yes, the string concatenation happens within a single complexity level if statement, not a for statement. I understand that it saves a single "new StringBuilder()" from being created.